docker/README.md

```diff
@@ -10,9 +10,10 @@
 
 * docker - Refer to the [official docker installation guide](https://docs.docker.com/install/)
 * git    - Refer to the [official git set-up guide](https://help.github.com/en/github/getting-started-with-github/set-up-git)
-* For base Ubuntu with GPU, install following nvidia container toolkit and driver- 
+* For base Ubuntu with GPU, install following nvidia container toolkit and driver-
   * [Nvidia container toolkit](https://github.com/NVIDIA/nvidia-docker#ubuntu-160418042004-debian-jessiestretchbuster)
   * [Nvidia driver](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-nvidia-driver.html)
+* NOTE - torchserve docker images have not been certified on windows native.
```

Comment:
NIT: I am not sure what "being certified" means in this context

Reply:
Made changes.